bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Decryption/DecryptWithShares.cs

```diff
@@ -14,29 +15,85 @@ public static class DecryptWithSharesExtensions
     public static PlaintextTally Decrypt(
         this CiphertextTally self,
         List<Tuple<ElectionPublicKey, TallyShare>> guardianShares,
+        List<GuardianChallengeResponse> challengeResponses,
```

Comment:
in this file, we now require that the challenge responses will be passed into the methods, which is necessary in order to propagate the proofs. There was an issue where the AccumulatedSelection was being assigned the proofs as part of the decryption process but they were not properly being added to the Decryption result. This way there's no way to miss it.